src/app/pages/vendor-dashboard/vendor-dashboard.component.ts

```diff
@@ -16,9 +17,20 @@ import {MatFormFieldModule} from '@angular/material/form-field';
 })
 export class VendorDashboardComponent implements OnInit {
 
-  constructor(private route: ActivatedRoute) {}
+  constructor(private route: ActivatedRoute, @Inject(MatDialog)private dialogRef : MatDialog) {}
 
   ngOnInit(): void {
     console.log('Called ngOnInit method');
   }
+  
+  openDialog(){
```

Comment:
This looks good but I think there will be some merge errors with variable and function names

Reply:
Yes, once we merge this PR into main, and you pull this @leeleonard936 you will want to use your changes not these. That way you can keep your UI for the dashboard and just call Liam's filter dialog